.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://art-bobcat.autodesk.com/artifactory/api/npm/autodesk-npm-virtual/
```

Comment:
Didn't we make this change before already?

Reply:
yes, but only on the jenkins branch.... which I think we should get rid of - all development should be on master to avoid things getting out of sync.

Reply:
https://github.com/DynamoDS/librarie.js/tree/develop

Reply:
it already is :( out of sync.

Reply:
@mjkkirschner Thx for the reminder!